libs/drape_frontend/gui/gui_text.cpp

```diff
@@ -421,12 +436,10 @@ MutableLabelHandle::MutableLabelHandle(uint32_t id, dp::Anchor anchor, m2::Point
 
 MutableLabelHandle::MutableLabelHandle(uint32_t id, dp::Anchor anchor, m2::PointF const & pivot,
                                        ref_ptr<dp::TextureManager> textures)
-  : TBase(id, anchor, pivot)
-  , m_textView(make_unique_dp<MutableLabel>(anchor))
-  , m_isContentDirty(true)
-  , m_textureManager(std::move(textures))
```

Comment:
Here the generated code will be faster than in the body of the constructor: move constructor will be used instead of default constructor + operator=

Reply:
Do not underestimate the power of optimizer :)

Reply:
And what are the reasons for _preliminary pessimization_?